src/lib.ts

```diff
@@ -550,19 +551,8 @@ export function canUse(item: Item): boolean {
  * @returns The thing specified or `null`
  */
 export function noneToNull<T>(thing: T): T | null {
```

Comment:
```suggestion
export function noneToNull<T extends MafiaClass>(thing: T): T | null {
```

Reply:
Not descriptively true, but perfectly prescriptively true yes.